src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -205,6 +207,10 @@ SaveInProgressIntro.propTypes = {
   downtime: PropTypes.object
 };
 
+SaveInProgressIntro.defaultProps = {
+  retentionPeriod: '60 days'
```

Comment:
Hey, nice use of default props! :raised_hands:
I keep forgetting this is a thing.